cmd/earthly/main.go

```diff
@@ -2379,31 +2379,22 @@ func (app *earthlyApp) actionAccountLogin(c *cli.Context) error {
 	}
 
 	if token != "" || pass != "" {
-		err := sc.DeleteCachedCredentials()
+		err := cc.DeleteCachedCredentials()
 		if err != nil {
 			return errors.Wrap(err, "failed to clear cached credentials")
 		}
-		sc.DisableSSHKeyGuessing()
+		cc.DisableSSHKeyGuessing()
 	} else if email != "" {
-		foundSSHKeys, err := sc.FindSSHAuth()
-		if err == nil {
-			if keys, ok := foundSSHKeys[email]; ok {
-				if len(keys) > 0 {
-					foundSSHKey := keys[0]
-					err := sc.SetLoginSSH(email, foundSSHKey)
-					if err != nil {
-						return err
-					}
-					fmt.Printf("Logged in as %q using ssh auth\n", email)
-					return nil
-				}
-			}
+		if err := cc.FindSSHCredentials(email); err != nil {
+			return errors.Wrap(err, "failed finding ssh keys")
```

Comment:
Why is this returning an error now?

Reply:
You're right it didn't return the error here before, and there's a switch case below that's set up to handle this error. Must've been a mistake, I'll fix that